v3/src/components/case-table/collection-table-spacer.tsx

```diff
@@ -9,13 +9,16 @@ import { measureText } from "../../hooks/use-measure-text"
 import { useVisibleAttributes } from "../../hooks/use-visible-attributes"
 import { logMessageWithReplacement } from "../../lib/log-message"
 import { IDataSet } from "../../models/data/data-set"
+import { symParent } from "../../models/data/data-set-types"
 // import { getNumericCssVariable } from "../../utilities/css-utils"
 import { preventAttributeMove, preventCollectionReorg } from "../../utilities/plugin-utils"
 import { t } from "../../utilities/translation/translate"
 import { kInputRowKey } from "./case-table-types"
 import { CurvedSpline } from "./curved-spline"
 import { useCollectionTableModel } from "./use-collection-table-model"
 
+const kRelationSelectedStrokeColor = '#66afe9' // blue
```

Comment:
Since this color also appears in the '.scss', is there a way to make a global out of it?

Reply:
Resolved in my code review tweaks commit.